var/spack/repos/builtin/packages/mofem-users-modules/package.py

```diff
@@ -86,6 +86,7 @@ class MofemUsersModules(CMakePackage):
     depends_on('mgis~python~fortran', when='+mgis')
     depends_on('tfel~python~python_bindings~fortran', when='+mgis')
 
+    depends_on('mofem-modules-manager', when='+modules-manager')
```

Comment:
that is reverse dependency, a circular problem; one depends on another and vice versa.